src/hooks/products-filters/use-filters.ts

```
@@ -0,0 +1,74 @@
+  const searchParams = useSearchParams();
+
+  // Form control
+  const form = useForm<FilterFormValues>({
+    defaultValues: {
+      occasionIds: [],
+      priceFrom: undefined,
+      priceTo: undefined,
+    },
+  });
+
+  const { watch, setValue, reset } = form;
```

ESLint

[error] 'reset' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)